release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1146,6 +1151,11 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-15-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-15-deprecation-sdn"]
+==== Deprecation of the OpenShift SDN network plugin
+
+OpenShift SDN CNI is deprecated as of {product-title} 4.14. As of {product-title} {product-version}, the network plugin is not an option for new installations. In a subsequent future release, the OpenShift SDN network plugin is planned to be removed and no longer supported. Red{nbsp}Hat will provide bug fixes and support for this feature until removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
```

Comment:
Small suggestion, up to you:
```suggestion
OpenShift SDN CNI is deprecated as of {product-title} 4.14. As of {product-title} {product-version}, the network plugin is not available for new installations. In a subsequent future release, the OpenShift SDN network plugin is planned to be removed and no longer supported. Red{nbsp}Hat will provide bug fixes and support for this feature until removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
```
I'm not sure that would even be accurate, so feel free to ignore.

Reply:
Thank you! I think "not an option" is most accurate and that language is approved by PM. I will leave this part as-is, but appreciate it!